cocos/core/value-types/bitmask.ts

```diff
@@ -95,7 +95,7 @@ export function BitMask<T> (obj: T): T {
  * @returns @en True if it is a BitMask, false else.
  * @zh 如果是 BitMask，返回 true；否则返回 false。
  */
-BitMask.isBitMask = (BitMaskType): any => BitMaskType && BitMaskType.hasOwnProperty('__bitmask__');
+BitMask.isBitMask = (BitMaskType): any => BitMaskType && Object.prototype.hasOwnProperty.call(BitMaskType, '__bitmask__');

```

Comment:
Why change it?

Reply:
Fix ESLint error:

```
Do not access Object.prototype method 'hasOwnProperty' from target object.eslint[no-prototype-builtins](https://eslint.org/docs/latest/rules/no-prototype-builtins)
```

ESLint doesn't suggest to use obj.hasOwnProperty on target object since target object may contain a same key `hasOwnProperty` to override the original behavior.